src/utils.ts

```diff
@@ -8,10 +8,11 @@ import { inspect, promisify } from 'util';
 
 const readFileAsync = promisify(readFile);
 
-type InputMode = 'translated' | 'reviewed' | 'proofread';
-type TxMode = 'translated' | 'reviewed' | 'reviewed2';
+type InputMode = 'untranslated' | 'translated' | 'reviewed' | 'proofread';
+type TxMode = 'developer' | 'translated' | 'reviewed' | 'reviewed2' ;
```

Comment:
:oops: extra space..

I also feel like we should update this to match all the current available modes
>developer | translator | reviewed | onlytranslated | onlyreviewed | sourceastranslation

https://docs.transifex.com/client/pull

Reply:
sorry for the extra space 😅 
If we give access to all of the mode, maybe can we remove the mapping, and take the transifex mode directly?

Reply:
yea.. wondering why there was such restriction in the first place..? maybe for typing? @kshahani

Reply:
Hi @tombertrand @kshahani 
any news concerning this?

Reply:
Yup just typing. We can continue to push this through as-is and circle back to add additional modes. ✅